pkg/logging/logger_test.go

```diff
@@ -0,0 +1,60 @@
+package logging
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestSetOutputs(t *testing.T) {
+	t.Run("default", func(t *testing.T) {
+		SetOutputs(nil, 0, 0)
+		if defaultLogger.Out != os.Stderr {
+			t.Error("Logger output should be stderr by default")
```

Comment:
Is it? looking at the switch case under logger.go, it should be either lumberjack or none (eventually io.Discard)?

Reply:
I kept the default behavior where if nothing is configured the default logger (logrus) is using stderr